### PR TITLE
[Chronos] Add install options in chronos-nightly docker image

### PR DESCRIPTION
**Pull request by hjzin**
### 1. Why the change?

Add install options in chronos-nightly docker image so that users can install chronos with necessary dependencies according to their needs, like the install options in the [documentation](https://bigdl.readthedocs.io/en/latest/doc/Chronos/Overview/chronos.html#install)
### 2. User API changes

Four build args are added when building the image, the new command could be:
```
sudo docker build \
    --build-arg model=pytorch \
    --build-arg auto_tuning=y \
    --build-arg hardware=single \
    --build-arg extra_dep=n \
     -t chronos-nightly:b1 .
```
The example for building chronos-nightly image can be seen in `BigDL/docker/chronos-nightly/README.md`.
### 3. Summary of the change 

1. Add four build args to choose necessary dependencies when building the image.
2. Update build command in `README.md`.

### CODE REVIEW

File: docker/chronos-nightly/README.md [hunk 1]
```diff
@@ -13,14 +13,49 @@ cp docker/chronos-nightly/Dockerfile ./Dockerfile
...
 sudo docker build \
     --build-arg http_proxy=http://<your_proxy_ip>:<your_proxy_port> \ #optional
     --build-arg https_proxy=http://<your_proxy_ip>:<your_proxy_port> \ #optional
+    --build-arg model=pytorch \
+    --build-arg auto_tuning=y \
+    --build-arg hardware=single \
+    --build-arg extra_dep=n \
     -t chronos-nightly:b1 . # You may choose any NAME:TAG you want.
 ```
 According to your network status, this building will cost **15-30 mins**.
```
Comment (TheaperDeng) [inline docker/chronos-nightly/README.md:61]: could we meet this build time?

File: docker/chronos-nightly/Dockerfile [hunk 2]
```diff
@@ -20,6 +20,16 @@ WORKDIR /opt/work
...
 
+# model: select from pytorch, tensorflow, prophet, arima, ml
+ARG model
+# auto_tuning: y (for yes) or n (for no)
+ARG auto_tuning
+# hardware: single or cluster
+ARG hardware
+# extra_dep: install extra dependencies(e.g. onnxruntime, jupyter, ...)
+#            y (for yes) or n (for no)
+ARG extra_dep
```
Comment (TheaperDeng) [inline docker/chronos-nightly/Dockerfile:31]: should we just give these parameters a default value? (pytorch|single|no tuning|no extra_dep)

**Review by TheaperDeng — COMMENTED** [review]
Please update this page as well https://github.com/intel-analytics/BigDL/blob/main/docs/readthedocs/source/doc/Chronos/Howto/docker_guide_single_node.md

File: docker/chronos-nightly/README.md (hunk 1 above)
Comment (hjzin) [reply docker/chronos-nightly/README.md:61]: Yeah. According to my test, when all options are specified, it take nearly 27 mins to finish, and when we only install chronos with pytorch, it takes 13-15 mins.

File: docker/chronos-nightly/Dockerfile (hunk 2 above)
Comment (hjzin) [reply docker/chronos-nightly/Dockerfile]: done.

**Review by TheaperDeng — APPROVED** [review]
LGTM, please generate a new document and put the link here.

**Comment by hjzin** [comment]
doc link: https://zhaojie-doc.readthedocs.io/en/install-option-docker/doc/Chronos/Howto/docker_guide_single_node.html

**Pull request merged**